### ISSUE TITLE
How to understand the meaning of post_rots and post_trans ?

### ISSUE THREAD

**Issue by jiayily** [nv-tlabs/lift-splat-shoot#18]
Hello, Thank you for your nice work. I really can't understand the meaning of post_rots and post_trans. Can you give an explaination about it ? Thank you again.

**Comment by e9190712** [comment]
Same problem.

**Comment by vesor** [comment]
I think it's the pre-processing transform which transform raw input images (1600,900) into network's input image (352, 128).

**Comment by tuan97ta** [comment]
I think `post_rots` and `post_trans` is `rots` and `trans` of image after augmentation.

**Comment by manueldiaz96** [comment]
> I think `post_rots` and `post_trans` is `rots` and `trans` of image after augmentation.

Yes, these matrices just account for the augmentations applied to the images. You can see this in the `get_geometry` method, when you modify the frustum coordinates with these augmentation matrices.
